growattServer/__init__.py

```diff
@@ -669,3 +669,23 @@ def update_ac_inverter_setting(self, serial_number, setting_type, parameters):
         }
         return self.update_inverter_setting(serial_number, setting_type, 
                                             default_parameters, parameters)
+    
+        def get_mix_inverter_settings(self, serial_number):
```

Comment:
The indenting looks off, so a sanity check. Isn't this the right indenting?
```suggestion
    def get_mix_inverter_settings(self, serial_number):
```